torch/_inductor/codegen/triton_utils.py

```diff
@@ -63,7 +63,12 @@ def signature_to_meta(
     }
 
 
-def config_of(args: List[KernelArgType], *, indices: Optional[List[int]] = None) -> Any:
+def config_of(
+    args: List[KernelArgType],
+    *,
+    indices: Optional[List[int]] = None,
+    add_equal_to_1: bool = False,
```

Comment:
Why not just add this always?

Reply:
My initial assumption was that, if this had not been handled before for the Inductor-generated Triton kernels, then there must have been some issue there. Let me try adding always and see how the CI responds.